api/v2/api.go

```diff
@@ -103,7 +102,8 @@ func NewAPI(alerts provider.Alerts, sf getAlertStatusFn, silences *silence.Silen
 
 	openAPI.Logger = func(s string, i ...interface{}) { level.Error(api.logger).Log(i...) }
 
-	api.Handler = openAPI.Serve(nil)
+	handleCORS := cors.New(cors.Options{}).Handler
```

Comment:
I guess one could also do `handler := cors.Default().Handler`, not sure if `cors.Options{}` always does the default?